src/bindings/native/index.js

```diff
@@ -0,0 +1,3 @@
+const native = require('./index.node');
```

Comment:
right now we load the wasm module [here ](https://github.com/MinaProtocol/mina/blob/d3069c6d0cc9ec89265bab5bb9d8f273d400dc1d/src/lib/crypto/kimchi_bindings/js/node_js/node_backend.js) to inject it into jsoo and load it again for usage in o1js itself [here](https://github.com/o1-labs/o1js-bindings/blob/649e6c34e773b8f37f9d0508da8ea57e2f4a63c7/js/node/node-backend.js)